test/sign_blob_test.sh

```diff
@@ -29,14 +29,19 @@ BLOB=verify-experimental-blob
 date > $BLOB
 cat $BLOB
 
+pass="$RANDOM"
+export COSIGN_PASSWORD=$pass
+./cosign generate-key-pair
+signing_key=cosign.key
+verification_key=cosign.pub
+
 echo "Sign the blob with cosign first and upload to rekor"
-$COSIGN_CLI sign-blob --output-certificate blob.cert --output-signature blob.sig $BLOB
```

Comment:
What is the reason for needing to change this? Do we lose coverage of the keyless flow with this change? I think both are useful as e2e tests.

Reply:
I may be able to keep this keyless depending on how GitHub OIDC mints tokens, though we'd need some extra work to make it work when run locally.

What's the intended usage for this test? I don't see it referenced in any workflows, so I added it to tests.yaml.

Reply:
I got this to work by treating the values of certificate-identity and certificate-oidc-issuer as regexps and then having the test pass in a match-all pattern.